working-groups/comms/charter.md

```diff
@@ -0,0 +1,61 @@
+# TAG ENV Working Group Communications Charter
```

Comment:
Should we also have a README-file in this folder that can provide easily accessible summary of what this working group is about, chairs/subchairs overview + contacts? We can then link to it in the WG table on the website in the first column?

Reply:
@guidemetothemoon @leonardpahlke Yes, agree. But we need the charter to be approved first.